pkg/datasource.go

```diff
@@ -142,13 +142,13 @@ func (plugin *GrafanaAzureDXDatasource) QueryData(ctx context.Context, req *back
 		return nil, err
 	}
 
-	email := "Noninteractive"
+	username := "Noninteractive"
```

Comment:
I'm not sure if it is more or less clean to move this into handleQuery, and pass plugincontext.user to the argument to handleQuery..

Reply:
Yeah I think so. I'll update. Thanks for feedback!